src/pug/pages/kutterjson.pug

```diff
@@ -13,6 +13,6 @@ append mainContent
       Robert made a live MJPEG preview running over a USB-C ethernet adapter to the \
       Ricoh Theta X.",
       "referenceLink": "https://community.theta360.guide/t/preview-mjpeg-stream-on-a-ricoh-theta-x-with-python-and-opencv/8919",
-      "image": "detail-sample.jpg"
+      "image": "kutter_project3.jpg"
```

Comment:
I'm getting a 404 file not found when I test the code.  Where did you put the `kutter_project3.jpg` file?

Did you put it in `public/assets/img/award`?  I don't see it there.

Reply:
I was putting images in live not public. I have placed the images in live now, it displays properly.